packages/grafana-data/src/transformations/transformers/calculateField.ts

```diff
@@ -693,7 +693,14 @@ export function getNameFromOptions(options: CalculateFieldTransformerOptions) {
     }
     case CalculateFieldMode.BinaryOperation: {
       const { binary } = options;
-      const alias = `${binary?.left?.matcher?.options ?? binary?.left?.fixed ?? ''} ${binary?.operator ?? ''} ${binary?.right?.matcher?.options ?? binary?.right?.fixed ?? ''}`;
+      const left = binary?.left?.matcher?.options ?? binary?.left?.fixed ?? '';
+      const right = binary?.right?.matcher?.options ?? binary?.right?.fixed ?? '';
+      const operator = binary?.operator ?? BinaryOperationID.Add;
+
+      let alias = '';
```

Comment:
I realize this function doesn't have any tests written yet. How do you feel about creating one with some simple use cases and this one, or just creating an issue so we can be sure to backfill them at some point?

Also, I realize the original code was structured this way, but what do you think about evaluating if theres a variable and returning blank sooner than this logic? Just so you dont do this and then find out theres a variable and throw it away.